parser-library/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
-add_library(pe-parser-library
-            buffer.cpp
-            parse.cpp)
+cmake_minimum_required(VERSION 3.1)
+project(pe-parser-library)
+
+add_library(${PROJECT_NAME} buffer.cpp parse.cpp)
+target_include_directories(${PROJECT_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
I added the public include folder here, so that other targets in the CMake project only need to link to the library to inherit it